tests/include/verifier.hpp

```diff
@@ -108,23 +108,15 @@ namespace gridtools {
             return out;
         }
 
-        template <class Expected, class DataStore, class Halos, class EqualTo = default_equal_to>
-        std::enable_if_t<storage::is_data_store<DataStore>::value &&
-                             is_view_compatible<Expected, DataStore::ndims>::value,
-            bool>
-        verify_data_store(Expected const &expected,
-            std::shared_ptr<DataStore> const &actual,
-            Halos const &halos,
+        template <class Expected, class Actual, size_t NDims, class EqualTo = default_equal_to>
+        bool verify(Expected const &expected,
```

Comment:
Maybe a less general name, just came up with `verify_functional`, but there might be better ones.
```suggestion
        bool verify_functional(Expected const &expected,
```

Reply:
verify_generic? see my other comment.